composer.json

```
@@ -72,7 +72,7 @@
 		"tomasvotruba/type-coverage": "1.0.0",
 		"typo3/cms-fluid-styled-content": "^12.4.26",
 		"typo3/coding-standards": "0.6.1 || 0.8.0",
-		"typo3/testing-framework": "7.1.1",
+		"typo3/testing-framework": "^8.2.7",
```

We're using fixed versions here to avoid build failures due to upgraded dependency versions without any changes on our side:

```suggestion
		"typo3/testing-framework": "8.2.7",
```

---

Done with f97be95558e243ba533bc56ae17e5a756e0cd097.